orbit-android/src/test/java/com/babylon/orbit/OrbitViewModelLifecycleTest.kt

```diff
@@ -52,8 +49,7 @@ class OrbitViewModelLifecycleTest {
     @Test
     fun `If I connect in onCreate I get disconnected in onDestroy`() {
         lateinit var orbitViewModel: OrbitViewModel<TestState, String>
-        val lifecycle = LifecycleRegistry(mock())
-        val lifecycleOwner = LifecycleOwner { lifecycle }
+        val lifecycleOwner = MockLifecycleOwner()
```

Comment:
Created a class because there is a cyclic dependency between `LifecycleOwner` and `LifecycleRegistry` which was not properly configured before.